database/migrations/2016_07_26_175053_create_reportback_table.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateReportbackTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('reportback', function (Blueprint $table) {
+            $table->increments('id')->comment('Primary key. Maps to rbid');
+            $table->string('user_id')->comment('The rogue users.id that correlates to their Northstar id');
```

Comment:
We've used `northstar_id` for this elsewhere so it might be nice to do that for consistency.